codegen/tests/simple/simple_test.go

```diff
@@ -398,6 +399,26 @@ func validQueryParamTest(t *testing.T, req GetStuffListRequest, query string) {
 	require.NoError(t, err)
 }
 
+func TestJsonValidateInput(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// Test request parameters
+		decoder := json.NewDecoder(r.Body)
+		req := new(GetStuffListRequest)
+		decodeErr := decoder.Decode(req)
```

Comment:
This line should result in a call through to the `UnmarshallJSON` method on `GetStuffListRequest`. I did a quick test but couldn't diagnose the issue.

Reply:
will have a look.

Reply:
Commented on the change here https://github.com/anz-bank/sysl-go/issues/108#issuecomment-651595445. Please review.